src/__tests__/amqp.module.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { Test } from '@nestjs/testing';
+import AmqpModule from './../amqp.module';
+import { ConfigModule } from 'nestjs-config';
+import { Injectable, Module } from '@nestjs/common';
+import { InjectAmqpConnection } from './../decorators';
+
+describe('Instance amqp module', () => {
+  it('Load module with an array of connection', async () => {
+    const module = await Test.createTestingModule({
+      imports: [
+        ConfigModule.load(),
+        AmqpModule.forRoot([
+          {
+            host: 'amqp://localhost:5672',
+          },
+          {
+            host: 'localhost',
+            port: 5672,
+            name: 'test',
+          },
+        ]),
+      ],
+    }).compile();
+
+    const connection1 = module.get<any>('amqpConnection_0');
+    const connectionTest = module.get<any>('amqpConnection_test');
+
+    expect(connection1).toBeTruthy();
+    expect(connectionTest).toBeTruthy();
+
+    connection1.close();
+    connectionTest.close();
+  });
+
+  it('Load module with singular connection', async () => {
+    const module = await Test.createTestingModule({
+      imports: [
+        ConfigModule.load(),
+        AmqpModule.forRoot({
+          host: 'amqp://localhost:5672',
+        }),
+      ],
+    }).compile();
+
+    const connection = module.get<any>('amqpConnection_default');
+
+    expect(connection).toBeTruthy();
+    connection.close();
+  });
+
+  it('Load module using env', async () => {
```

Comment:
this test is not actually testing the functionality with the `.env` file? i can't see that file.

Reply:
At the moment there isn't a config file for this, the default is `localhost:5672` from connection